fix_appdata.patch

```diff
@@ -0,0 +1,42 @@
+diff --git a/share/appdata/howl.appdata.xml b/share/appdata/howl.appdata.xml
+index 5a131eb..6d9ae28 100644
+--- a/share/appdata/howl.appdata.xml
++++ b/share/appdata/howl.appdata.xml
+@@ -1,9 +1,12 @@
+ <?xml version="1.0" encoding="UTF-8"?>
+ <component type="desktop">
+- <id>howl.desktop</id>
++ <id>io.howl.Editor</id>
```

Comment:
You can drop this change, it is updated when using rename-appdata-file https://github.com/flathub/io.howl.Editor/blob/cffffcea4201ecc2df47052c5540bda116ccd211/io.howl.Editor.yaml#L6